### ISSUE TITLE
Safari (Mac 13.1.1, IPAD OS 13.5.1, IOS 13.5.1): Screen freezes when other user starts screen sharing

### ISSUE THREAD

**Issue by informatikTirol** [jitsi/jitsi-meet#6965]
The (Main) Screen freezes when an other user shares his screen (only the user which shares the screen is frozen) on Safari (Mac 13.1.1, IPAD OS 13.5.1, IOS 13.5.1). Tested with https://meet.jit.si/ and with latest nightly build jitsi-meet_4696 on AWS.

With the latest nightly build the screen freezes only if the user with screen sharing enabled shares also the video camera. 

On reload you the screen gets unfrozen. If then the user stops and then starts screen sharing again the screen for that user in Safari gets frozen again.

Also interesting: In the little preview window on the right you see the frozen screen sharing screen and the not frozen webcam of the user as transparent overlay

With Google Chrome it works.

**Comment by informatikTirol** [comment]
@saghul

**Comment by ludi81** [comment]
I have exactly the same problem. Do you have a solution?

**Comment by nathankurtyka** [comment]
tl;dr - Screen sharing is 100% broken in Safari (on any device)

**Comment by pgenderson** [comment]
It's also broken on Mac Chrome.
